### PR TITLE
30 left side navbar

### PR DESCRIPTION
**Pull request by DLiSWE**
left navbar
Screenshot 2023-01-08 063416
Screenshot 2023-01-08 063355

### CODE REVIEW

**Review by raylu511 — COMMENTED** [review]
Can you align the icons and the text

**Comment by DLiSWE** [comment]
Screenshot 2023-01-08 161613

**Comment by Matt-Tan15** [comment]
you should align all the icons to the left vertically

**Comment by DLiSWE** [comment]
Screenshot 2023-01-08 175033

File: src/shared/components/NavSidebar/NavSidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import {
+  RssFeedRounded,
+  MailRounded,
+  LiveHelpRounded,
+  BookmarkRounded,
+} from "@mui/icons-material";
+
+type menuItem = {
+  name: string;
+  icon: SvgIconProps;
```
Comment (raylu511) [inline src/shared/components/NavSidebar/NavSidebar.tsx:18]: I think you can type this as a JSX.Element

File: src/shared/components/NavSidebar/NavSidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+};
+
+type NavSidebarProps = {
+  theme: any;
+};
+
+const NavSidebar = ({ theme }: NavSidebarProps) => {
+  const menuItems: menuItem[] = [
```
Comment (raylu511) [inline src/shared/components/NavSidebar/NavSidebar.tsx:26]: nit: maybe you can create a iconStylingProps

**Review by raylu511 — APPROVED** [review]

**Review by raylu511 — DISMISSED** [review]

**Comment by Matt-Tan15** [comment]
nit: I'd align the C in chats with the other items it's slightly out of place

**Review by Matt-Tan15 — APPROVED** [review]

**Review by emir4ik-code — APPROVED** [review]
Looks good!

**Pull request merged**